tests/sentry/issues/auto_source_code_config/test_process_event.py

```diff
@@ -154,7 +167,19 @@ def test_dry_run_platform(self) -> None:
             assert len(code_mappings) == 1
             assert code_mappings[0].stacktrace_root == "foo/"
             assert code_mappings[0].source_path == "src/foo/"
-            assert not RepositoryProjectPathConfig.objects.exists()
+
+    def test_extension_is_not_included(self) -> None:
+        frame_filename = "foo/bar.tbd"
+        file_in_repo = "src/foo/bar.tbd"
+        platform = "foo"
+        self.event = self.create_event([{"filename": frame_filename, "in_app": True}], platform)
+
+        with patch(f"{CODE_ROOT}.task.supported_platform", return_value=True):
+            # No extensions are supported, thus, we can't generate a code mapping
+            self._process_and_assert_no_code_mapping([file_in_repo])
```

Comment:
The code mapping does not get generated since the `tbd` file extension has not yet been added to the list of supported extensions.